reactor-netty-core/src/main/java/reactor/netty/tcp/TcpServer.java

```diff
@@ -127,6 +127,10 @@ public TcpServer metrics(boolean enable) {
 		return super.metrics(enable);
 	}
 
+	public TcpServer metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new TcpServerConfig.MicrometerTcpServerMetricsRecorder(prefix));
+	}
+
```

Comment:
Is this needed? One can use 
`reactor.netty.transport.Transport#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)` with `MicrometerChannelMetricsRecorder`